.github/workflows/checks.yml

```
@@ -0,0 +1,24 @@
+name: Run XO and Prettier
+on:
+  pull_request:
```

Do we want to make this consistent with the Course Search checks and ensure that they run on merge to `dev` and `main`, and also that they can be run manually? @ce666-york
```suggestion
on:
  push:
    branches:
      - dev
      - main
  pull_request:
  workflow_dispatch:
```